### ISSUE TITLE
Hero color issue

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1629]
Looks like the default reverse is now black (used to be white).

@aschmoe and I discussed briefly but adding here so we can discuss and resolve if need be.

Example: https://help.proudcity.com/accessibility

**Comment by kevindherman** [comment]
Here's an example on Jackson: https://www.jacksonms.gov/american-disabilities-act/

@lukefretwell @aschmoe
